app/controllers/users_controller.rb

```diff
@@ -58,7 +66,6 @@ def user_params
     :password,
     :phone,
     :pinterest_name,
-    :privacy_settings,
```

Comment:
K, let's add positions to search. Didn't mean to remove privacy settings..that was work from a different PR. Will fix.